chukwuemekachm/utility_manager_web
# Prof: adds view when link is invalid

#### What does this PR achieve?
- extends the forgot ConfirmResetPassword to have a view when the link
is invalid
- extracts query params value extractor to new util class
- replaces EmailSentSuccess to reuseable AuthMessage component
- adds docs for new component added

#### Any background context?
- None

#### Manual Testing Steps
- When you navigate to `/reset-password/fsd?expiresAt=2021-01-01`(expiresAt has a value in the future) you should see the form but when you navigate to `/reset-password/fsd?expiresAt=2020-01-01`( expiresAt has a value in the past) you should invalid link message

#### Screenshots:
<img width="1440" alt="Screenshot 2020-01-12 at 12 47 50 PM" src="https://user-images.githubusercontent.com/38565349/72218287-c6075a00-3539-11ea-9597-b93db919851c.png">
![Uploading Screenshot 2020-01-12 at 12.47.27 PM.png…]()

#### Relevant Trello Tickets
- [Invalid reset password link](https://trello.com/c/76heq3KN/60-d-1-implement-link-expired-page)
